src/routes.js

```diff
@@ -4,16 +4,24 @@ import { AppPlaceholder } from './presentational-components/shared/loader-placeh
 import { routes } from '../package.json';
 import useChrome from '@redhat-cloud-services/frontend-components/useChrome';
 
+import { exploreServerlessQuickStart } from './test/quick-start/explore-serverless-quickstart';
+import { explorePipelinesQuickStart } from './test/quick-start/explore-pipeline-quickstart';
+
 const Groups = lazy(() => import('./smart-components/group/groups'));
 const Roles = lazy(() => import('./smart-components/role/roles'));
 const Users = lazy(() => import('./smart-components/user/users'));
 const MyUserAccess = lazy(() => import('./smart-components/myUserAccess/MUAHome'));
 const AccessRequests = lazy(() => import('./smart-components/accessRequests/accessRequests'));
+const ResourcesCatalog = lazy(() => import('./smart-components/ResourcesCatalog'));
 
 export const Routes = () => {
+  const experimentalChrome = localStorage.getItem('experimental:useChrome') === 'true';
+  const chrome = useChrome();
+  React.useEffect(() => {
+    experimentalChrome && chrome.setQuickStarts([exploreServerlessQuickStart, explorePipelinesQuickStart]);
```

Comment:
In App's entry point, can provide API to add their quick start files through `chrome.setQuickStarts`